readthedocs/projects/migrations/0138_remove_old_fields.py

```diff
@@ -0,0 +1,47 @@
+# Generated by Django 4.2.16 on 2024-11-20 12:35
+
+from django.db import migrations
+from django_safemigrate import Safe
+
+
+class Migration(migrations.Migration):
+    safe = Safe.after_deploy
+
+    dependencies = [
+        ('projects', '0137_use_generic_root_selector'),
+    ]
+
+    operations = [
+        migrations.RemoveField(
+            model_name='addonsconfig',
+            name='doc_diff_root_selector',
```

Comment:
I think we talked about migrating this data into `options_root_selector`?

Reply:
I've already checked those projects and their custom selectors used for docdiff and our logic will work for them 👍🏼